inspec-core.gemspec

```diff
@@ -0,0 +1,43 @@
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'inspec/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'inspec-core'
+  spec.version       = Inspec::VERSION
+  spec.authors       = ['Dominik Richter']
+  spec.email         = ['dominik.richter@gmail.com']
+  spec.summary       = 'Just InSpec'
+  spec.description   = 'Core InSpec, local support only. See `inspec` for full support.'
+  spec.homepage      = 'https://github.com/chef/inspec'
+  spec.license       = 'Apache-2.0'
+
+  spec.files = %w{README.md MAINTAINERS.toml MAINTAINERS.md LICENSE
+                  inspec-core.gemspec Gemfile CHANGELOG.md} +
+               Dir.glob('{bin,docs,examples,lib}/**/*', File::FNM_DOTMATCH)
+                  .reject { |f| File.directory?(f) || f =~ /aws|azure|gcp/ }
```

Comment:
We should (eventually) make this regex more obvious - possibly make it a variable or env var.  This will be a brittle point.